tools/ldb_test.py

```diff
@@ -547,5 +554,25 @@ def testColumnFamilies(self):
         # non-existing column family.
         self.assertRunFAIL("get cf3_1 --column_family=four")
 
+    def testIngestExternalSst(self):
+        print "Running testIngestExternalSst..."
+
+        # Dump, load, write external sst and ingest it in another db
+        dbPath = os.path.join(self.TMP_DIR, "db1")
+        self.assertRunOK("batchput --db=%s --create_if_missing x1 y1 x2 y2 x3 y3 x4 y4" % dbPath, "OK")
```

Comment:
80-char

Reply:
Looks like `make format` does not check python files. I'll manually fix.